api-report/runtime-definitions.api.md

```diff
@@ -62,8 +62,8 @@ export type FluidDataStoreRegistryEntry = Readonly<Partial<IProvideFluidDataStor
 
 // @public
 export enum FlushMode {
-    Automatic = 0,
-    Manual = 1
+    Immediate = 0,
```

Comment:
So, given these are numbers, this is not breaking change, and compat UTs should pass, right? Just confirming.

Reply:
Yes. It works :)